docs/go-tutorial-prerequisites.md

```diff
@@ -12,4 +12,4 @@ Make sure you have [Go](https://golang.org/doc/install) installed. These tutoria
 
 ## Temporal server
 
-Download, install, and run the [Temporal server](/docs/install-temporal-server) via docker-compose. It is easy to do and you can keep it running in the background while you build applications.
+Download, install, and run the Temporal server via `docker-compose`. You can find Temporal server setup instructions here: [https://docs.temporal.io/docs/install-temporal-server](https://docs.temporal.io/docs/install-temporal-server). It is easy to do and you can keep it running in the background while you build applications.
```

Comment:
nit:
```suggestion
Download, install, and run the Temporal server via `docker-compose`. You can find Temporal server setup instructions here: [https://docs.temporal.io/docs/install-temporal-server](/docs/install-temporal-server). It is easy to do and you can keep it running in the background while you build applications.
```
I guess you can do a relative link for the actual link - and show the full URL as the text.